packaging/nuget/NServiceBus.Persistence.Sql.MsBuild.targets

```diff
@@ -1,12 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <ScriptDirectory>$(ProjectDir)$(IntermediateOutputPath)NServiceBus.Persistence.Sql\</ScriptDirectory>
```

Comment:
after moving to the new vs2017 csproj system it did not correctly evaluate that property